hanksTodo/README.md

```diff
@@ -1,30 +1,70 @@
-# Hankstodo demo application
+# HanksTodo android demo application
 
-Follow the following steps to build and run hanksTodo demo app.
+### About the App
+The hanksTodo app allows users to create multiple todo lists. Users can maintain multiple todo items within a todo list. Once you are done with a todo item or a list, you can go ahead and delete them.
+
+### Amino MicroServices
+1. **TodoListManager** :
+    Second class in the app to implement the Microservice and makes use of the Default Deployment Manager(DM).
+    
+2. **TodoList** :
+    This class implements the MicroService and makes use of multi-DM. The combination of DMs used is DHT and ConsensusRSM.
```

Comment:
Rephrase it to -
This class implements the MicroService and uses a combination of DHT and ConsensusRSM DMs.

Reply:
Done.